### DISCUSSION TITLE
Custom component

### DISCUSSION THREAD

**Discussion by RobbieFR** [hneemann/Digital#1282] [Q&A]
Hello, 
I was able to change the size and the color of a custom component (subcircuit) i made and i saw in the documentation that we should be able to create our really own component looks using svg, if any of you can describe the process to do so it would be helpfull !

PS : My teacher said i would have an extra bonus point if i could put my face as one of the component lol

**Comment by hneemann** [comment]
Go to the circuit-specific settings and select the "User-defined" shape. Click on the "Template" button to create an svg file as a template for your design. This file can then be edited with a tool such as Inkscape or similar. However, the pins may only be moved, not deleted. Once this is done, the file can be imported using the "Import" button. 
Unfortunately, bit-maps embedded in the svg are not imported. So you cannot simply insert a photo.  :smile:
